mongo-init.js

```diff
@@ -1,15 +1,26 @@
 // Create Databases
 const dbPims = db.getSiblingDB('pims');
 const dbRemsAdmin = db.getSiblingDB('remsadmin');
```

Comment:
Is there any way to make these commands run the next one even if there is a failure? When I ran the init script on a database I already had, it would error out on lines to create users that already existed. Not a showstopper, but would be nice to have.